authentication/tests.py

```diff
@@ -103,7 +106,7 @@ class SendVerificationEmailTest(TestCase):
     def setUp(self):
         self.client = APIClient()
         self.password = BaseUserManager().make_random_password()
-        self.user = AppUser.objects.create_user(email='email@email.com',username='testuser',password=self.password)
+        self.user = AppUser.objects.create_user(email='gmail@gmail.com',username='testuser',password=self.password, real_username="user", real_email = "email@email.com")
```

Comment:
can you add this to BaseTestCase in the utils folder? since it also uses the user object

Reply:
I am removing the additional columns, so it doesn't need to be changed